packages/firestore/rollup.config.browser.js

```diff
@@ -34,12 +38,49 @@ export default [
     }
   },
   {
-    input: pkg.esm2017,
-    output: { file: pkg.module, format: 'es', sourcemap: true },
-    plugins: util.es2017ToEs5Plugins(/* mangled= */ true),
+    input: {
+      index: 'index.ts',
+      memory: 'index.memory.ts',
+      bundle: 'index.bundle.ts'
+    },
+    output: {
+      dir: 'dist/esm2017',
+      format: 'es',
+      sourcemap: true
+    },
+    plugins: [
+      alias({
+        entries: [
+          {
+            find: /^(.*)\/export$/,
+            replacement: `$1\/dist/prebuild.js`
+          }
+        ]
+      }),
+      ...util.es2017Plugins('browser', /* mangled= */ false)
```

Comment:
This should be mangled.

Reply:
As discussed offline, this should be false.